zivid_camera/src/settings_generator.cpp

```diff
@@ -159,6 +196,10 @@ class DynamicReconfigureCfgGenerator
     {
       return SettingsNode::validRange().min();
     }
+    if constexpr (IsEnumSetting<SettingsNode>::value)
+    {
+      return *SettingsNode::validValues().begin();
```

Comment:
So validValues will here return a `std::set`, and it's implicit that the first element of the set is the default value (until the default value can be gotten from the camera)?

Reply:
Yes we gotta pick something, so we pick the first.